src/index.tsx

```diff
@@ -258,8 +258,5 @@ const signInWithRedirect = (): void => {
   }
 };
 
-const getCookieValue = (name: string): string => {
-  return (
-    document.cookie.match('(^|;)\\s*' + name + '\\s*=\\s*([^;]+)')?.pop() || ''
-  );
-};
+const getCookieValue = (name: string): string =>
```

Comment:
you can put () after the => so it looks better if you want, enclosing the code there

Reply:
Nah I really need to get back on task with my class project lol

Reply:
ex. `const hi = () => ('hi!');`

Reply:
If you want you can change it idc

Reply:
says the person that said I should put curly braces after throwing an error because prettier made it 2 lines

Reply:
;)

Reply:
Fiiiinnneee

Reply:
Bruh prettier gets rid of it

Reply:
lol